test.js

```diff
@@ -148,13 +189,20 @@ test('toVFile.writeSync', function (t) {
     st.end();
   });
 
+  t.throws(
+    function () {
+      vfile.writeSync(invalidFilePath);
+    },
+    /ENOENT/,
+    'should throw on files that cannot be written'
+  );
+
   t.end();
 });
 
 test('toVFile.write', function (t) {
   var filePath = 'fixture.txt';
-
-  t.plan(4);
```

Comment:
No need for `.plan` on this level; this line just makes adding more test cases difficult 😃